{
  "CVE Code": "CVE-2022-34981",
  "Published": "2022-07-22T14:17:30",
  "Updated": "2024-08-03T09:29:17.380Z",
  "Description": "The PyCrowdTangle package in PyPI before v0.0.1 included a code execution backdoor inserted by a third party.",
  "ProblemTypes": [
    "n/a"
  ],
  "Affected": [
    {
      "vendor": "n/a",
      "product": "n/a",
      "versions": [
        "n/a"
      ]
    }
  ],
  "Reference": [
    "http://pypi.doubanio.com/simple/request",
    "https://pypi.org/project/PyCrowdTangle/",
    "https://github.com/UPB-SS1/PyCrowdTangle/issues/1"
  ]
}